src/parser/index.js

```diff
@@ -59,7 +59,7 @@ const parseNode = (node) => {
   };
 
   if (parsedNode.note) {
-    parsedNode.note = parsedNode.note.replace('if you think this can be improved in any way  please say', '');
+    parsedNode.note = parsedNode.note.replace('', 'if you think this can be improved in any way  please say');
```

Comment:
I reverted this change, cause it was correct as it was before. Here we're stripping the notes we had with links to GitHub issues.